### DISCUSSION TITLE
Lamellar with specific direction of the parent crystal

### DISCUSSION THREAD

**Discussion by fischertim1** [neperfepx/neper#386] [Q&A] [answered]
Can anybody give me an example of the new option "crysdir(<crysdir_x>,<crysdir_y>,<crysdir_z>)" in the "lamellar(w=<width>[,v=<normal>][,pos=<pos>])" command.
Many thank!

**Comment by rquey** [comment, marked as answer]
Lamellae of normal directions parallel to the 100 crystal directions:
```
neper -T -n 10::from_morpho -morpho "voronoi::lamellar(w=0.1,v=crysdir(1,0,0))" -o lam
neper -V lam.tess -datacellcol "scaleid(1)" -print img
```

![image](https://user-images.githubusercontent.com/2659376/153429559-12fb1bb1-e470-457b-b091-c76b55c15d71.png)
